web_widget_numeric_step/static/src/scss/numeric_step.scss

```diff
@@ -0,0 +1,29 @@
+
+.input_numeric_step {
+    width: 71% !important;
+}
+
+.btns_numeric_step {
+    width: 27% !important;
+    button {
+        height: 25px;
+    }
+}
+
+/* Turn Off Number Input Spinners */
+input{
+    /* Firefox */
+    -moz-appearance: textfield;
+
+    /* Chrome */
+    &::-webkit-inner-spin-button {
```

Comment:
On chromium I am still able to see both sets of the buttons.

Reply:
Yep you're right.. can't fond the way in chromium. Every blog post about this problem are solving this for Firefox/Opera/Chrome but it seems to be different for Chromium..